app/src/colorpalettewidget.cpp

```diff
@@ -17,16 +17,20 @@ GNU General Public License for more details.
 #include "colorpalettewidget.h"
 #include "ui_colorpalette.h"
 
-#include <cmath>
+// Standard libraries
+#include "cmath"
```

Comment:
minor thing, but usually system header uses angle brackets

Reply:
Ah right, a typo. It should indeed be angle brackets.